src/dattobd.c

```diff
@@ -2409,7 +2560,15 @@ static int snap_mrf(struct request_queue *q, struct bio *bio){
 		bio_endio(bio, -EBUSY);
 		return 0;
 	}
-	
+
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 3, 0))
+	struct bio_set *bio_split = bioset_create(BIO_POOL_SIZE, 0);
```

Comment:
im not sure where this is getting freed. I see that if it is not needed it gets freed in snap_queue_split(), but if it is used I think it will become leaked.